playbooks/conferences.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Conferences
+description: Attending or speaking at a conference
+---
+
+# Conferences
+
+We encourage engineers to attend and speak at conferences. It is a great way to stay up-to-date with the industry,
+be inspired by challenges others are facing, develop personal career, and connect with the community. Here are some
+guidelines:
+
+## Attending Conferecens
+
+- We aim for each engineer to be able to attend a conference every year, subject to budget. The conference should
+  be related to individual's role or Artsy's challenges/technologies.
```

Comment:
```suggestion
  be related to the individual's role or Artsy's challenges/technologies.
```

Reply:
In the past we’ve agreed to send engineers to courses/workshops that are great for the individual’s growth, but less directly for their current role. An example would be an intensive 5 day design course. Do we consider that beneficial to their _future_ role, an ‘Artsy challenge’, or something separate?